tests/message/unittest_message.py

```diff
@@ -1,29 +1,31 @@
 # Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
 # For details: https://github.com/PyCQA/pylint/blob/main/LICENSE
 
-from typing import Dict, ValuesView
+from typing import ValuesView
 
+from pylint.interfaces import HIGH
 from pylint.message import Message
 from pylint.message.message_definition import MessageDefinition
+from pylint.typing import MessageLocationTuple
 
 
 def test_new_message(message_definitions: ValuesView[MessageDefinition]) -> None:
     def build_message(
-        message_definition: MessageDefinition, location_value: Dict[str, str]
+        message_definition: MessageDefinition, location_value: MessageLocationTuple
     ) -> Message:
         return Message(
             symbol=message_definition.symbol,
             msg_id=message_definition.msgid,
-            location=[
-                location_value["abspath"],
-                location_value["path"],
-                location_value["module"],
-                location_value["obj"],
-                location_value["line"],
-                location_value["column"],
-            ],
+            location=(
+                location_value.abspath,
+                location_value.path,
+                location_value.module,
+                location_value.obj,
+                location_value.line,
+                location_value.column,
+            ),
```

Comment:
Why is this necessary? Wasn't the whole point so that you could do `location=location_value`?

Reply:
Sometimes after your reviews I wonder if I was even thinking when submitting a PR... 😅

Reply:
I missed this one, damn ! 😄